### ISSUE TITLE
False positive for document.hasFocus() on Chrome 27

### ISSUE THREAD

**Issue by MrTimscampi** [amilajack/eslint-plugin-compat#304]
The compat/compat rule throws the following error with Chrome 27 as a target:

```
document.hasFocus() is not supported in Chrome 27
```

According to caniuse, document.hasFocus() is supported on all versions of Chrome: https://caniuse.com/#feat=mdn-api_document_hasfocus

**Comment by amilajack** [comment]
Fixed in https://github.com/amilajack/eslint-plugin-compat/pull/323. You can get the latest fixes by installing the latest version of the plugin: `yarn add --dev eslint-plugin-compat@latest`.
